.github/ISSUE_TEMPLATE/offboarding-request.md

```diff
@@ -1,15 +1,14 @@
 ---
 name: Offboarding request
-about: To inform the Platform team when a team VSP or VFS team member has left and
-  needs offboarding.
+about: To inform the Platform team when a VSP or VFS team member has left and needs offboarding.
 title: Offboarding of [individual]
 labels: analytics-insights, Offboarding, operations
 assignees: ''
 
 ---
 
 # Instructions
-Fill out name of individual and *Description* section below.
+Fill out name of individual and **Description** section below. Note: This form only handles deactivation from the DSVA (Office of CTO @ VA) Slack instance. For Lighthouse Slack removal, visit [this page](https://confluence.devops.va.gov/display/VAExternal/LHDI+Offboarding+Process).
```

Comment:
Rather then referencing the Confluence, you should reach out to the @department-of-veterans-affairs/lighthouse-owls

Reply:
```suggestion
Fill out name of individual and **Description** section below. Note: This form only handles deactivation from the DSVA (Office of CTO @ VA) Slack instance. For Lighthouse Slack removal, post a message in the [#lighthouse-delivery-platform-support](https://lighthouseva.slack.com/archives/C03UA9MV1EH) and @lighthouse-di-enablement team for assistance.
```
Feel free to adjust the wording if anything sounds off but until we get our ticketing system in, reaching out to the owls team in the lighthouse-delivery-platform-support channel is the current way to get help.